docs/lint_errors.md

```diff
@@ -27,13 +31,18 @@ The following files are suggested but not a hard requirement. If they are missin
 * `conf/base.config`
   * A `conf` directory with at least one config called `base.config`
 
+Additionally, the following files must not be present:
+
+* `Singularity`
 
 ## Error #2 - Docker file check failed ## {#2}
+
 Pipelines should have a files called `Dockerfile` in their root directory.
 The file is used for automated docker image builds. This test checks that the file
 exists and contains at least the string `FROM` (`Dockerfile`).
```

Comment:
Are we checking that the `Dockerfile` countains at least just `FROM`?
Or are we checking that it countains `FROM nfcore/base`? maybe with a specific tag?

Reply:
Yep. Just `FROM` currently:
https://github.com/nf-core/tools/blob/615c6a57d0cddefb7f38bac1c4ea83704bfd9214/nf_core/lint.py#L318